winch/codegen/src/isa/aarch64/regs.rs

```diff
@@ -162,5 +160,13 @@ pub(crate) fn callee_saved() -> SmallVec<[Reg; 9]> {
         xreg(26),
         xreg(27),
         xreg(28),
+        vreg(8),
```

Comment:
There's some subtlety here in that only the lower half of the `v8`..`v15` registers are callee-saved. In Cranelift we overapproximate on both sides (when saving, save the whole register; when generating a callsite, assume whole register is clobbered), and then have an optimization where if our own function's ABI and the callee's ABI are the same then at callsites we ignore these for clobbers. I haven't thought through all the implications with Winch's regalloc model but I just wanted to make sure we've considered this here?

Reply:
Agreed, as discussed offline, I've added size information to the callee-saved registers.